deploy/db-writer.yaml

```diff
@@ -0,0 +1,293 @@
+                  key: aws_access_key_id
+                  optional: true
+            - name: INSIGHTS_RESULTS_AGGREGATOR__CLOUDWATCH__AWS_SECRET_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: cloudwatch
+                  key: aws_secret_access_key
+                  optional: true
+          image: ${IMAGE}:${IMAGE_TAG}
+          livenessProbe:
+            failureThreshold: 10
+            httpGet:
+              path: "${IRDW_API_PREFIX}metrics"
+              port: 9000
+              scheme: HTTP
+            initialDelaySeconds: 60
+            periodSeconds: 60
+            successThreshold: 1
+            timeoutSeconds: 60
+          readinessProbe:
+            failureThreshold: 10
+            httpGet:
+              path: "${IRDW_API_PREFIX}metrics"
+              port: 9000
+              scheme: HTTP
+            initialDelaySeconds: 60
+            periodSeconds: 60
+            successThreshold: 1
+            timeoutSeconds: 60
+          resources:
+            limits:
+              cpu: ${DVO_WRITER_CPU_LIMIT}
+              memory: ${DVO_WRITER_MEMORY_LIMIT}
+            requests:
+              cpu: ${DVO_WRITER_CPU_REQUEST}
+              memory: ${DVO_WRITER_MEMORY_REQUEST}
+    database:
+      # the DB name should match to app-interface DB name without specifying environment
+      # https://gitlab.cee.redhat.com/service/app-interface/-/blob/ddd85c2ad79b40047391405b2d909eb38667bc43/data/services/insights/ccx-data-pipeline/namespaces/stage-ccx-data-pipeline-stage.yml#L60
+      name: ccx-data-pipeline
+      version: 15
+    kafkaTopics:
+      - replicas: 3
+        partitions: 1
+        topicName: ${INCOMING_TOPIC}
+      - replicas: 3
+        partitions: 1
+        topicName: ${PAYLOAD_TRACKER_TOPIC}
+      - replicas: 3
+        partitions: 1
+        topicName: ${DEAD_LETTER_QUEUE_TOPIC}
+- kind: Service
+  apiVersion: v1
+  metadata:
+    annotations:
+      prometheus.io/path: /metrics
+      prometheus.io/port: "9000"
+      prometheus.io/scheme: http
+      prometheus.io/scrape: "true"
+    name: dvo-writer-prometheus-exporter
+    labels:
+      app: dvo-writer
+  spec:
+    ports:
+      - name: web
+        port: 9001
+        protocol: TCP
+        targetPort: 9001
+    selector:
+      app: dvo-writer
+      pod: dvo-writer-instance
+
+parameters:
+- name: PG_PARAMS
+  value: sslmode=require
+- name: KAFKA_BOOTSTRAP_HOST
+  required: true
+  value: mq-kafka
+- name: KAFKA_BOOTSTRAP_PORT
+  required: true
+  value: "29092"
+- name: KAFKA_TIMEOUT
+  value: 300s
+- name: INCOMING_TOPIC
+  value: ccx.dvo.results
+  required: true
+- name: DEAD_LETTER_QUEUE_TOPIC
+  value: ccx.aggregator.dead.letter.queue
+  required: true
+- name: PAYLOAD_TRACKER_TOPIC
+  value: platform.payload-status
+  required: true
+- name: DVO_WRITER_SERVICE_NAME
+  value: dvo-writer
+  required: true
+- name: GROUP_ID
+  value: ccx_data_pipeline_app
+  required: true
+- name: IRDW_API_PREFIX
```

Comment:
IRDW means insights results DB writer

Reply:
oh, true, I will use another name then

Reply:
fixed in https://github.com/RedHatInsights/insights-results-aggregator/pull/1906/commits/12f69fb3d9d768cb952d89d0da32138fd1023271